moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummary.js

```diff
@@ -7,7 +7,12 @@ import ProjectSummaryStatusUpdate from "./ProjectSummaryStatusUpdate";
 import { Grid, CardContent, CircularProgress } from "@material-ui/core";
 import ApolloErrorHandler from "../../../../components/ApolloErrorHandler";
 
-import ProjectSummaryProjectEntity from "./ProjectSummaryProjectEntity";
+/*
+  Error Handler and Fallback Component
+*/
+import ProjectSummaryMapFallback from "./ProjectSummaryMapFallback";
+import { ErrorBoundary } from "react-error-boundary";
```

Comment:
My bad because I swept this out from under you. I removed `react-error-boundary` in this file and one other file in [this PR](https://github.com/cityofaustin/atd-moped/pull/915) since it wasn't a widely used pattern.

Reply:
no worries, thanks for the heads up!